resources/windows/msi/service.wxs.erb

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+<!-- Some ERB Here to include the set of service files -->
+
+  <?include $(sys.SOURCEFILEDIR)include/puppet.wxi ?>
```

Comment:
Is this a file that's going to be installed from the project? Were these going to be parameterized?